.env

```diff
@@ -0,0 +1,2 @@
+PGDATABASE=nc_news_test;
+PGDATABASE=nc_news;
```

Comment:
You should have 2 .env files, one for development with the name of your development database and one for your test database. Both of these should be git ignored